File: src/employeemanager/model/Position.java
package employeemanager.model;

public class Position {
    private int positionId;
    private String title;

    public Position(int positionId, String title) {
        this.positionId = positionId;
        this.title = title;
    }

    public int getPositionId() {
        return positionId;
    }

    public String getTitle() {
        return title;
    }
}

File: Part 3/EmployeeManager/models/Position.cs
namespace EmployeeManager.Models
{
    public class Position
    {
        public int PositionId { get; set; }
        public string Title { get; set; }

        public Position(int positionId, string title)
        {
            PositionId = positionId;
            Title = title;
        }
    }
}